client/js/components/toggle.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import injectSheet from 'react-jss'
+
+const Toggle = ({ classes, toggle, onClick }) => (
+	<div
+		className={classes.toggle}
+		onClick={onClick}
+		style={{ color: toggle ? '#F48271' : '#D8D8D8' }}
+	>
+		<input type="checkbox" className={classes.toggleSwitch} value={toggle} />
+		<label className={classes.toggleLabel}>
+			<span
+				className={classes.toggleButton}
+				style={{ transform: toggle ? 'translateX(70%)' : '' }}
+			/>
+		</label>
+	</div>
+)
+
+const styles = {
+	toggle: props => ({
```

Comment:
We should be able to remove the `props` and make this into an object, rather than a function now.